test/hotspot/jtreg/runtime/Monitor/MonitorWithDeadObjectTest.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2022, 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ *
+ */
+
+/*
+ * @bug 8320515
+ * @summary This test checks that ObjectMonitors with dead objects don't
+ *          cause asserts, crashes, or failures when various sub-systems
+ *          in the JVM find them.
+ * @library /testlibrary /test/lib
+ * @modules jdk.management
+ */
+
+/*
+ * @requires os.family != "windows" & os.family != "aix"
```

Comment:
I guess we can fix the AIX usage later.

Reply:
This test tests a very specific platform-independent bug. Making it work for other platforms than Linux was done when it was easy to do so. I can't test AIX and don't know all the quirks of that OS, so I don't think it is worth for me to spend the time trying to make this test work for AIX.